src/parser/monk/mistweaver/modules/spells/essences/WayOfTheCrane.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import Analyzer from 'parser/core/Analyzer';
+import StatisticBox from 'interface/others/StatisticBox';
```

Comment:
probably worth using a different box for this seeing as StatisticBox is deprecated, I don't know how important that deprecation warning is though, maybe using an ItemStatistic? (e.g. voidstone module for the dropdown)

Reply:
So happy I wasn't the only know that didn't realize that happened

Reply:
idk when it happened but I remember seeing it in the StatisticBox source file when I implemented my voidstone module so I added the dropdown functionality to interface/statistic/Statistic (which ItemStatistic uses)

Reply:
Hmm... I wonder which box is correct here. Since AzeritePowerStatistic isn't fully accurate but is closer than item imo 🤔